apps/api/src/server/app/case-officer/case-officer-review.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable complexity */
+// import { validationResult } from 'express-validator';
+
+const reviewIncomplete = function (body) {
+	return ((
+		body.Reason.applicationPlanningOficersReportMissingOrIncorrect == true ||
+		body.Reason.applicationPlansToReachDecisionMissingOrIncorrect  == true ||
+		body.Reason.policiesStatutoryDevelopmentPlanPoliciesMissingOrIncorrect  == true ||
+		body.Reason.policiesOtherRelevanPoliciesMissingOrIncorrect  == true ||
+		body.Reason.policiesSupplementaryPlanningDocumentsMissingOrIncorrect   == true ||
+		body.Reason.siteConservationAreaMapAndGuidanceMissingOrIncorrect  == true ||
+		body.Reason.siteListedBuildingDescriptionMissingOrIncorrect  == true ||
+		body.Reason.thirdPartyApplicationNotificationMissingOrIncorrect  == true ||
+		body.Reason.thirdPartyApplicationNotificationMissingOrIncorrectListOfAddresses  == true ||
+		body.Reason.thirdPartyApplicationNotificationMissingOrIncorrectCopyOfLetterOrSiteNotice  == true ||
+		body.Reason.thirdPartyApplicationPublicityMissingOrIncorrect  == true ||
+		body.Reason.thirdPartyRepresentationsMissingOrIncorrect  == true ||
+		body.Reason.thirdPartyAppealNotificationMissingOrIncorrect  == true ||
+		body.Reason.thirdPartyAppealNotificationMissingOrIncorrectListOfAddresses  == true ||
+		body.Reason.thirdPartyAppealNotificationMissingOrIncorrectCopyOfLetterOrSiteNotice  == true ) ?
+		false : true );
```

Comment:
I think it might be an awesome exercise to look up if there is a more efficient way to do this

Reply:
You could build an array and iterate to check these conditions instead of a massive || like this. Also we should use strict check with === instead ==.

Reply:
Again, refactoring in separate tasks